src/components/SummaryList.tsx

```diff
@@ -0,0 +1,59 @@
+import classNames from "classnames";
+import PropTypes, { arrayOf } from "prop-types";
```

Comment:
```suggestion
import PropTypes from "prop-types";
```

Autocomplete import?